docs/CMIP7/guidance_for_users.md

```diff
@@ -5,11 +5,233 @@ title: CMIP7 Guidance for Data Users
 
 # CMIP7 Guidance for Data Users
 
-## 1.  Experiment Design
-## 2.  Model output specification
-## 3.  Accessing Model output 
-## 4.  Terms of use and citation requirements
-## 5.  Model and experiment documentation
-## 6.  Reporting suspected errors
-## 7.  Registering published work based on CMIP7
-## 8.  CMIP7 organisation and governance
\ No newline at end of file
+This page is designed to inform data users on key CMIP7 concepts and tools. It is a landing page to redirect them to the proper resources to learn more.
+
+
+
+## 1.  Accessing CMIP7 data
+
+CMIP7 model output is available through a distributed data archive developed and operated by the Earth System Grid Federation (ESGF). The data are hosted on a collection of nodes located at modeling centers or data centers across the world.
+
+
+There are 2 options to access the data:
+
+ 1. **MetaGrid** ([LLNL][metagridllnl], [DKRZ][metagriddkrz], [ORNL][metagridornl], [CEDA][metagridceda])
+
+    Easy-to-use web interface to search and download ESGF data. It provides access through http downloads, wget scripts, OPeNDAP URLs and Globus transfers. The data can be accessed through any of the CMIP7 web interfaces linked above, which enable users to search across the entire distributed archive as if it were all centrally located.
+
+ 2. [**ESGpull**][esgpull]
+
+    A python library that allows the user to interface with the ESGF search API. It handles scanning, downloading and updating datasets, files and queries from ESGF.
+
+This page will be updated as other access routes become available.
+
+
+## 2.  Terms of use, citations and registration requirements
+
+To enable modeling groups and others who support CMIP7 to demonstrate its impact (and secure ongoing funding), you are required to cite and acknowledge those who have made CMIP7 possible. When using CMIP7 data, you must: 
+
+ 1. **Acknowledge CMIP7.**
+
+    In the Acknowledgment section, please insert the following text:
+
+    >We acknowledge the World Climate Research Programme, which, through its Working Group on Coupled Modelling, coordinated and promoted CMIP7. We thank the climate modeling groups for producing and making available their model output, the Earth System Grid Federation (ESGF) for archiving the data and providing access, and the multiple funding agencies who support CMIP7 and ESGF.
+
+ 2. **Cite the specific dataset(s) used.**
+
+    Please include a citation in the form of:
+
+        Authors/Data Creators (publication year): Title. Version YYYYMMDD. Earth System Grid Federation. DOI.
+        
+    e.g. 
+
+        Swart et al. (2019): CCCma CanESM5 model output prepared for CMIP6 ScenarioMIP. Version 20190429. Earth System Grid Federation. https://doi.org/10.22033/ESGF/CMIP6.1317. 
```

Comment:
We probably need guidance on how to get the title, version and DOI (or a tool that can just automatically do this if given the file headers)

Reply:
I see we have guidance below. Reading that, I think a tool is more essential (if you use 50 datasets, doing that process by hand is very painful)

Reply:
Agreed. In an ideal world, the citation would be a global attrs, but I assume this is not possible at this point...